### PR TITLE
Feat/d4 g 214 design magic link email

### PR DESCRIPTION
**Pull request by Baboo7**
## Proposed changes

ticket : https://ogre-d4g.atlassian.net/browse/D4G-214

Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.

## Types of changes

---

What types of changes does your code introduce to ogre ?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, renaming)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Documentation Update (if none of the other choices apply)
- [ ] Other (please describe):

## Checklist

---

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

offseason_ogre/blob/master/CONTRIBUTING.md) doc

- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation and modify the README.md (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] If the feature changes current behavior, reasons why your solution is better.

## What is the current behavior?

---

-
-
-

## What is the new behavior?

---

-
-
-

## Further comments

---

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

## Where has this been tested?

---

Docker version: …
Operating System: …
node version: …
postgresql version: …

image

### CODE REVIEW

File: packages/server/src/modules/notifications/services/mails.ts [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+  if (!config) {
+    throw new Error(`Could not find config for email ${templateName}`);
+  }
+
+  await doSendEmail({
+    to,
+    templateData,
+    templateId: config.templateId,
+    asmGroupId: config.asmGroupId,
```
Comment (ghost) [inline packages/server/src/modules/notifications/services/mails.ts:46]: @Baboo7 c'est quoi ce `asmGroupId `?

**Review by ghost — APPROVED** [review]
@Baboo7 c'est trop beau! 🤩

**Review by louis-sanna-dev — APPROVED** [review]
@Baboo7 j'approuve aussi!

File: packages/server/src/modules/notifications/services/mails.ts (hunk 1 above)
Comment (Baboo7) [reply packages/server/src/modules/notifications/services/mails.ts:46]: Ce sont les groupes de désabonnement pour les mails marketing. Ça permet aux utilisateurs de dire qu'ils se désabonnent des mails produits, ceux associés à leur compte... Pour le moment on n'utilise pas les groupes

**Pull request merged**